package.json

```diff
@@ -41,17 +44,19 @@
     "ember-disable-prototype-extensions": "^1.0.0",
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-export-application-global": "^1.0.4",
-    "ember-try": "^0.2.0",
```

Comment:
I think @sandersky was adding `ember-try` to all of our repos, I assume we want to keep it here